### PR TITLE
Edits to Node Operation docs from external (Natasha)

### PR DESCRIPTION
**Pull request by 10thfloor**

### CODE REVIEW

File: docs/content/concepts/flow-concepts/transactions.mdx [hunk 1]
```diff
@@ -24,7 +24,7 @@ A Transaction is the smallest unit of computation performed by the Flow network.
...
 
-Consensus nodes are responsible for forming blocks containing collection guarantees. Finalized blocks are transmitted to execution nodes who then contact the collector nodes that guaranteed each collection for the contents of those collections.
+Consensus nodes are responsible for forming blocks containing collection guarantees. Finalized blocks are transmitted to execution nodes who then contact the Collection nodes that guaranteed each collection for the contents of those collections.
```
Comment (psiemens) [inline docs/content/concepts/flow-concepts/transactions.mdx:27]:
```suggestion
Consensus nodes are responsible for forming blocks containing collection guarantees. Finalized blocks are transmitted to execution nodes who then contact the collection nodes that guaranteed each collection for the contents of those collections.
```

File: docs/content/node-operation/node-bootstrap.mdx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Node bootstrap
-sidebar_title: Node bootstrap
+title: Node Bootstrap
+sidebar_title: Node Bootstraping
```
Comment (10thfloor) [inline docs/content/node-operation/node-bootstrap.mdx:3]:
```suggestion
sidebar_title: Node Bootstrapping
```

File: docs/content/concepts/flow-concepts/slashing.mdx [hunk 3]
```diff
@@ -68,11 +68,11 @@ Transitioning to a rigorous staking protocol in which all slashable conditions a
 
 **Adjudication:** Consensus nodes evaluate the content of the collection, if the collection is found invalid, the Collection Node who signed the content is slashed.
 
-**1.3 Double Collection Proposal Witness (DCPW):** Collection nodes of a cluster run a mini consensus inside the cluster to decide on a collection, which requires collectors to propose the collection and aggregate votes from others. During the collection consensus, if a Collection Node proposes more than one proposal, any other Collection Node inside the cluster can report a Double Collection Proposal Witness (including both proposals).
+**1.3 Double Collection Proposal Witness (DCPW):** Collection nodes of a cluster run a mini consensus inside the cluster to decide on a collection, which requires Collection nodes to propose the collection and aggregate votes from others. During the collection consensus, if a Collection Node proposes more than one proposal, any other Collection Node inside the cluster can report a Double Collection Proposal Witness (including both proposals).
```
Comment (vishalchangrani) [inline docs/content/concepts/flow-concepts/slashing.mdx:71]:
```suggestion
**1.3 Double Collection Proposal Witness (DCPW):** Collection nodes of a cluster run a mini consensus inside the cluster to decide on a collection, which requires collection nodes to propose the collection and aggregate votes from others. During the collection consensus, if a collection node proposes more than one proposal, any other collection node inside the cluster can report a Double Collection Proposal Witness (including both proposals).
```

Comment (vishalchangrani) [reply docs/content/concepts/flow-concepts/slashing.mdx:71]: lower case node type ? as we discussed on the slack thread earlier

**Review by vishalchangrani — APPROVED** [review]
lgtm

**Comment by laynelafrance** [comment]
Node Minimums:
Collection: 250,000
Consensus: 500,000
Execution: 1,250,000
Verification: 135,000
cc @10thfloor

File: docs/content/concepts/flow-concepts/slashing.mdx (hunk 3 above)
Comment (10thfloor) [reply docs/content/concepts/flow-concepts/slashing.mdx:71]: I think we'll wait, because there are multiple instances of the previous casing.

**Pull request merged**